### PR TITLE
uhd: pybind RFNoC graph, streamers

### PR DESCRIPTION
**Pull request by marcusmueller**
@bastibl  [noticed](https://matrix.to/#/!UHlAIKBoAIYHCdpjhE:gnuradio.org/$tFPDSWZrDkG4KvtADKkZx9og4jCdd41TLNCSZaTHMAg?via=gnuradio.org&via=matrix.org&via=atrz.de) that these are used in GRC, but not actually pybound in 3.9

### CODE REVIEW

**Comment by marcusmueller** [comment]
This needs a bit of unit testing, I'd say. Not quite sure what to test without hardware. @mbr0wn, ideas?

File: gr-uhd/python/uhd/bindings/CMakeLists.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@ list(APPEND uhd_python_files
     usrp_block_python.cc
     usrp_sink_python.cc
     usrp_source_python.cc
+    rfnoc_graph_python.cc
```
Comment (mormj) [inline gr-uhd/python/uhd/bindings/CMakeLists.txt:13]: does rfnoc_rx,tx_streamer need to go here as well?

Comment (marcusmueller) [reply gr-uhd/python/uhd/bindings/CMakeLists.txt:13]: yes!

**Comment by marcusmueller** [comment]
@mbr0wn unlike on my machine, this fails in CI due to missing `uhd/rfnoc/noc_block_base.hpp`

File: gr-uhd/python/uhd/bindings/docstrings/rfnoc_graph_pydoc_template.h [hunk 2]
```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 202 Marcus Müller
```
Comment (mbr0wn) [inline gr-uhd/python/uhd/bindings/docstrings/rfnoc_graph_pydoc_template.h:2]: Is 202 when you started your Bachelor's degree? :)

**Review by mbr0wn — COMMENTED** [review]
> This needs a bit of unit testing, I'd say. Not quite sure what to test without hardware. @mbr0wn, ideas?

UHD 4.1 will include an RFNoC-hardware-simulator, but it isn't very advanced (yet).

**Review by mbr0wn — APPROVED** [review]

File: gr-uhd/python/uhd/bindings/docstrings/rfnoc_graph_pydoc_template.h (hunk 2 above)
Comment (marcusmueller) [reply gr-uhd/python/uhd/bindings/docstrings/rfnoc_graph_pydoc_template.h:2]: :D prehistoric burn!

**Pull request merged**